### ISSUE TITLE
Add Development Guidelines

### ISSUE THREAD

**Issue by oswalpalash** [oswalpalash/AndroidLiquid#14]
Need to document the development guidelines for the project.

**Comment by SmoothHacker** [comment]
Hey, can I help? Do you need a CONTRIBUTING.md for the project?

**Comment by oswalpalash** [comment]
@SmoothJazzMusic yes you can. Yeah we do need CONTRIBUTING.md also, we need to lay down the scope of improvement on the project.

**Comment by SmoothHacker** [comment]
Sure, what do you need for the development guidelines?
